File: lessons42576/solution.java
import java.util.Arrays;

class Solution {
    public String solution(String[] participant, String[] completion) {
        int c_size = completion.length;
        
        // O(nlogn) 
        Arrays.sort(participant);
        Arrays.sort(completion);
        
        for(int i=0;i<c_size;i++){
            if(!participant[i].equals(completion[i])){
                return participant[i];
            }
        }
        
        return participant[c_size];
    }
}

File: lessons42576/solution.js
function solution(p, c) {
    var size = c.length;
    p.sort((a,b)=>(a < b ? -1 : (a > b ? 1 : 0)));
    c.sort((a,b)=>(a < b ? -1 : (a > b ? 1 : 0)));
    for(var i=0;i<size;i++){
        if(p[i]!=c[i]){
            return p[i];
        }
    }   
    return p[size];
}
